packages/zowe-explorer/src/Profiles.ts

```diff
@@ -557,6 +572,50 @@ export class Profiles extends ProfilesCache {
         return profileType;
     }
 
+    public async createZoweSchema(zoweFileProvider: IZoweTree<IZoweTreeNode>) {
+        try {
+            ImperativeConfig.instance.loadedConfig = {
+                defaultHome: path.join(os.homedir(), ".zowe"),
+                envVariablePrefix: "ZOWE",
+            };
+
+            // If there are no workspaces already opened, default to the cliHome
+            const rootPath = vscode.workspace.workspaceFolders?.[0].uri.fsPath || ImperativeConfig.instance.cliHome;
+            // true-case-path may or may not be needed. Better safe than sorry : )
```

Comment:
@crawr should we have a second pop-up option after configPick is chosen for the user to choose team or global?  Automatically assigning to team if a workspace is open sets it up to be pushed to SCM and that may not what the user wants.

Reply:
Yes, I see your point and I like the idea of choosing global/project level config. But now I am wondering if it should be a second pop-up here? Another option would be during the selection of the `+` button there should be three choices

1. `+ Create a New Connection to z/OS`
2. `+ Create a New Global Configuration File`
3. `+ Create a New Project Configuration File` (only show this if a workspace is open)

Not sure if this will be more confusing, though

Reply:
I agree with @JillieBeanSim. Let's do it this way

`+ Create new Configuration File` >> `Global` or `Project`

Reply:
Ok, I will add a second prompt 👍